### DISCUSSION TITLE
how to zoom entire vs code window

### DISCUSSION THREAD

**Discussion by Mbittu00** [vhqtvn/VHSCode-Android#204] [Ideas]
How could I zoom whole vscode window

**Comment by pfontana77** [comment]
I have also problem with the DPI of the editor on Fold 4, it's too small. There Is a way to jncrease It?

**Reply by trashyms** [reply]
Hello.
Any solution would be more than okay.

**Comment by vhqtvn** [comment]
hold the semi-transparent FN button appear when you touch at editor screen, tap settings, and then change UI Scale.

**Reply by trashyms** [reply]
Perfect! Thank for the reply.
